sdk/tests/conformance/glsl/misc/00_test_list.txt

```diff
@@ -108,6 +108,6 @@ uniform-location-length-limits.html
 --min-version 1.0.3 struct-nesting-of-variable-names.html
 --min-version 1.0.3 struct-specifiers-in-uniforms.html
 --min-version 1.0.3 struct-unary-operators.html
---min-version 1.0.4 --max-version 1.9.9 ternary-operator-on-arrays.html
+--min-version 1.0.4 ternary-operator-on-arrays.html
```

Comment:
Is removing the --max-version correct? I'm pretty sure that the test will be run with a WebGL 2.0 context if it's removed.